cmd/mms/commands.go

```diff
@@ -121,3 +129,29 @@ func productReceiver(event *mms.ProductEvent) error {
 	fmt.Println(event)
 	return nil
 }
+
+func createExecutableCallback(filepath string) func(event *mms.ProductEvent) error {
+	_, err := exec.LookPath(filepath)
+	fmt.Println(filepath)
+	if err != nil {
+		panic(fmt.Errorf("command executable not found, %s", err.Error()))
+
+	}
+
+	return func(event *mms.ProductEvent) error {
```

Comment:
Can the ProductEvent fields be available to the command line somehow?

Reply:
8293ea6 Adds product name as arg[1]